spec/system/index_spec.rb

```diff
@@ -4,8 +4,20 @@
 
 RSpec.describe "Index page", type: :feature do
 
-    it "includes the Smartflix title" do
+
+    it "includes the Smartflix header" do
         visit "/"
         expect(page).to have_text("Smartflix")
     end
+
+   context "when titles are given" do
+
+    it "includes the shows titles" do 
+      titles.first(5).each do |title|
+        visit "/" 
```

Comment:
I think we can visit "/" once and asserts the titles.

Reply:
Yes! thank you Simon!